.gitignore

```diff
@@ -15,5 +15,8 @@
 *.iml
 **/*.idea
 
+*/out
+*.class
```

Comment:
You don't need this, if out/ folder will be ignored

Reply:
I understand. I tried but  I can not remove out directory from git.